Binary/Loader/StreamLoader.php

```diff
@@ -60,7 +60,7 @@ public function find($path)
          * The error suppression is solely to determine whether the file exists.
          * file_exists() is not used as not all wrappers support stat() to actually check for existing resources.
          */
-        if (($this->context && !$resource = @fopen($name, 'r', null, $this->context)) || !$resource = @fopen($name, 'r')) {
+        if (($this->context && !$resource = @fopen($name, 'rb', null, $this->context)) || !$resource = @fopen($name, 'rb')) {
```

Comment:
aparently `b`  is active by default and thats why the cs-fixer does not want to have it? https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/4011

Reply:
https://www.php.net/manual/en/function.fopen.php

> For portability, it is strongly recommended that you always use the 'b' flag when opening files with fopen().

Reply:
hm. i guess then we should go with the php doc and have the "b". it should not hurt, even if the php doc might be outdated and all current php versions now have "b" on. that means we have to disable that check in the .php_cs.dist file